numpy/lib/_function_base_impl.py

```diff
@@ -5743,13 +5750,25 @@ def append(arr, values, axis=None):
     array([[1, 2, 3],
            [4, 5, 6],
            [7, 8, 9]])
+
     >>> np.append([[1, 2, 3], [4, 5, 6]], [7, 8, 9], axis=0)
     Traceback (most recent call last):
         ...
     ValueError: all the input arrays must have same number of dimensions, but
     the array at index 0 has 2 dimension(s) and the array at index 1 has 1
     dimension(s)
 
+    >>> a = np.array([1, 2], dtype=int)
+    >>> b = []
+    >>> c = np.append(a, b)
+    >>> c
+    array([1., 2.])
+    >>> c.dtype
+    float64
+
+    Default dtype for empty lists is `float64` thus making the output of dtype
```

Comment:
Move this comment to above the example
```suggestion
    Default dtype for empty ndarrays is `float64` thus making the output of dtype
```